src/applications/pre-need/tests/components/SupportingDocumentDescription.unit.spec.jsx

```diff
@@ -25,7 +25,7 @@ describe('<SupportingDocumentsDescription>', () => {
     );
 
     tree
-      .find('button')
+      .find('a')
```

Comment:
The test changes are because [this PR](https://github.com/department-of-veterans-affairs/component-library/pull/182) which changed `<AdditionalInfo>` to render an `<a>` instead of a `<button>` got included in version 3.2.1 of `component-library`.